server/datastore/mysql/migrations/tables/20230913141311_AddResolvedInVersionToSoftwareCVE.go

```diff
@@ -0,0 +1,27 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230913141311, Down_20230913141311)
+}
+
+func Up_20230913141311(tx *sql.Tx) error {
+	stmt := `
+		ALTER TABLE software_cve
+		ADD COLUMN resolved_in_version VARCHAR(50)
```

Comment:
long enough?

Reply:
`software.version` is `varchar(255)`, maybe overkill and `varchar(50)` is a-ok?

Also, let's add `COLLATE utf8mb4_unicode_ci DEFAULT NULL`

Reply:
UPDATE: We saw some linux kernel/module versions being larger than 50 in our Dogfood environment, let's match `software.version`'s length (255).